crates/dekaf/src/session.rs

```diff
@@ -257,112 +257,77 @@ impl Session {
             .with_topics(topics))
     }
 
-    // Lists all read-able collections as Kafka topics. Omits partition metadata.
+    // Lists all read-able collections as Kafka topics
     async fn metadata_all_topics(&mut self) -> anyhow::Result<Vec<MetadataResponseTopic>> {
```

Comment:
the metadata_all_topics behavior changed here to actually report partitions since we have them cached